projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -144,6 +153,10 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * @hidden @internal
      */
     public defaultFallbackGroup = 'Other';
+    /**
+     * @hidden @internal
+     */
+    public caseSensitive = false;
```

Comment:
Please create an object `filtertingOptions` that for the moment will hold only `caseSensitive` option. For the moment we will not expose this as an input.

Reply:
I have created the filteringOptions object an an interface IComboFilteringOptions